myRental/migrations/20180320173008_addUsersAndRequest.js

```diff
@@ -0,0 +1,31 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTableIfNotExists('REQUESTS', (table) => {
```

Comment:
Why are you using `createTableIfNotExists` here? Wouldn't the table not exist before this runs? The way knex migrations work is that there is a migrations table kept in the database tracking which migrations have run, so this won't be ran everytime we run `knex migrate:latest`. It will just run any migrations that have not run already.